CHANGELOG.md

```diff
@@ -76,15 +77,9 @@ The types of changes are:
 
 ### Added
 
-<<<<<<< HEAD
-- Added new Wunderkind Consent Saas Connector [#2600](https://github.com/ethyca/fides/pull/2600)
-- Added new Sovrn Email Consent Connector [#2543](https://github.com/ethyca/fides/pull/2543/)
-- Log Fides version at startup [#2566](https://github.com/ethyca/fides/pull/2566)
-=======
```

Comment:
@allisonking I think looking at main these might still be correct now? A little unsure if we keep overwriting things somewhere since it sounded like these might have been fixed 🤷🏽

Reply:
yeah it sounds like it was fixed in main whew